lib/canvas_link_migrator/link_resolver.rb

```diff
@@ -18,9 +18,14 @@
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
 require "active_support/core_ext/object"
+require "rack"
```

Comment:
I'm ignorant, why do we require rack now?

Reply:
It was already required in this file, but we didn't hit it until we had a file map to use.

Reply:
Line 194 and 200 reference Rack.